### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by franpog859** [BartekPog/PK-notatki#1]
It would be great to translate it into English and maybe to Chinese!

**Comment by BartekPog** [comment]
Yes! But unfortunately due to lacking time I will not be able to work on this subject in near future.
